static/js/rule.js

```diff
@@ -234,7 +233,7 @@ function guessDisplayFields(data, view) {
   else {
     var keys = ['gc', 'cov', 'len'];
     var avails = [];
-    for (var i = 1; i < keys.length; i++) {
+    for (var i = 0; i < keys.length; i++) {
```

Comment:
What is the purpose of this? I guess my original rationale is that the first column is always contig indentifier (`id` or something), so the iteration may start from the second column.

Reply:
Yes, I understood that the iteration should start from the second column. But for some reason the web interface is not taking gc or coverage as the x and y if I put i = 1. However, if I put i = 0, the web interface is considering gc and coverage as x and y.